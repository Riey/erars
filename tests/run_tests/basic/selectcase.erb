@SYSTEM_TITLE
CALL FOO, 0
CALL FOO, 1
CALL FOO, 2
CALL FOO, 3
CALL FOO, 4

@FOO(ARG)
SELECTCASE ARG
    CASE 0
        PRINTL FOO
    CASE 1 TO 2, 3
        PRINTL BAR
    CASEELSE
        PRINTL BAZ
ENDSELECT
