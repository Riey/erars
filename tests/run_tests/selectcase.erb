@SYSTEM_TITLE
SELECTCASE 1
    CASE 0
        PRINT FOO
    CASE 1 TO 2
        PRINT BAR
    CASEELSE
        PRINT BAZ
ENDSELECT
