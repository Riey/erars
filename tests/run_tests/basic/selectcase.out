FOO
BAR
BAR
BAR
BAZ
